04-concept-api/references/attribute.yml

```diff
@@ -187,17 +187,32 @@ methods:
       title: Check if value is of type datetime (Local)
       description: Returns true if the attribute value is of type datetime. Otherwise, returns false.
     java:
-      <<: *method-isdouble
+      <<: *method-isdatetime
       method: attribute.isDateTime();
       returns:
         - boolean
     javascript:
-      <<: *method-isdouble
+      <<: *method-isdatetime
       method: attribute.isDateTime();
       returns:
         - boolean
     python:
-      <<: *method-isdouble
+      <<: *method-isdatetime
       method: attribute.is_datetime()
       returns:
         - boolean
+
+  - method:
+    common: &method-toJSON
+      title: Retrieve an attribute as a JSON.
+      returns:
+        - JSON with fields `type`, `value_type` and `value`
+    java:
+      <<: *method-toJSON
+      method: attribute.toJSON();
+    javascript:
+      <<: *method-toJSON
+      method: await attribute.toJSON();
+    python:
+      <<: *method-toJSON
+      method: attribute.toJSON()
```

Comment:
here as well, should be `to_json()` and the nodejs is `toJSONRecord`

Reply:
Fixed in https://github.com/vaticle/docs/pull/678/commits/7eb27634059a4ee8c9f38ab9eed9e2b0345f72c4